models/DataObject/Traits/ContextPersistenceTrait.php

```diff
@@ -25,6 +25,65 @@
  */
 trait ContextPersistenceTrait
 {
+    protected function prepareMyCurrentRelations(
+        Localizedfield|AbstractData|\Pimcore\Model\DataObject\Objectbrick\Data\AbstractData|Concrete $object,
```

Comment:
Maybe we could consistently use `Objectbrick\Data\AbstractData` and `Fieldcollection\Data\AbstractData` in this file for better readability 😊